lessons/ro/chapter_9.yaml

```diff
@@ -0,0 +1,177 @@
+    Fiecare program sau bibliotecă Rust este o *cutie*.
+
+    Fiecare cutie este alcătuit dintr-o ierarhie de *module*.
+
+    Fiecare cutie are un modul rădăcină.
+
+    Un modul poate conține variabile globale, funcții, structuri, trăsături sau chiar alte module!
+
+    În Rust nu există o corespondență 1 la 1 între fișiere și ierarhia de module. Trebuie să construim arborele de module în mod explicit și manual în codul nostru.
+- title: Scrierea unui Program
+  content_markdown: |
+    Un program are un modul rădăcină într-un fișier numit `main.rs`.
+- title: Scrierea unei Biblioteci
+  content_markdown: |
+    O bibliotecă are un modul rădăcină într-un fișier numit `lib.rs`.
+- title: Referințierea la alte Module și Cutii
+  code: >-
+    https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&code=use%20std%3A%3Af64%3A%3Aconsts%3A%3API%3B%0A%0Afn%20main()%20%7B%0A%20%20%20%20println!(%22Welcome%20to%20the%20playground!%22)%3B%0A%20%20%20%20println!(%22I%20would%20love%20a%20slice%20of%20%7B%7D!%22%2C%20PI)%3B%0A%7D%0A
+  content_markdown: >
+    Elementele din module pot fi refererențiate cu calea completă a acestora.
+    `std::f64::consts::PI`.
+
+
+    O modalitate mai simplă este cuvântul cheie **use**. Acesta ne permite să specificăm
+    anumite elemente din modulele pe care dorim să le
+
+    să folosim în tot codul nostru fără o cale completă. Spre exemplu `use
+    std::f64::consts::PI`
+
+    îmi permite să folosesc doar identificatorul `PI` în funcția mea principală.
+
+
+    **std** este o cutie a **bibliotecii standard** din Rust, care este plină de
+    date utile
+
+    structuri și funcții utile pentru interacțiunea cu sistemul de operare.
+
+
+    Un repertoriu de căutare a cutiilor create de comunitate poate fi găsit la adresa
+    [https://crates.io](https://crates.io/).
+- title: Referința la mai multe elemente
+  content_markdown: |
+    Mai multe elemente pot fi menționate într-o singură cale de modul, astfel:
+
+    ```rust
+    use std::f64::consts::{PI,TAU}
+    ```
+
+    Ferris doesn't eat TAU, he only eats PI.
+- title: Crearea de Module
+  content_markdown: |
+    Când ne gândim la cod, ne imaginăm de obicei o ierarhie de fișiere organizate
+    în directoare. Rust vă permite să creați module strâns legate de fișierul dvs.
+    structura fișierelor.
+
+    În Rust există două moduri de a declara un modul. De exemplu, un modul
+    `foo` poate fi reprezentat ca:
+      * un fișier numit `foo.rs`
+      * un director numit `foo` cu un fișier `mod.rs` în interior
+- title: Ierarhia Modulelor
+  content_markdown: >
+    Un modul poate depinde de un alt modul. Pentru a stabili o relație
+
+    între un modul și submodulul său, trebuie să scrieți în modulul părinte:
+
+
+    ```rust
+
+    mod foo;
+
+    ```
+
+
+    Declarația de mai sus va căuta un fișier numit `foo.rs` sau `foo/mod.rs`.
+    și
+
+    va insera conținutul acestuia în interiorul unui modul numit `foo` în acest domeniu de aplicare.
```

Comment:
What is *domeniu de aplicare*?

Reply:
Previous commit touched on this by calling it "scope" directly.

Reply:
Got it, I think we used *domeniu de vizibilitate* in previous chapters.

Reply:
Updated the translation to reflect that.